include/circt/Dialect/SV/Structure.td

```diff
@@ -0,0 +1,90 @@
+//===- Structure.td - SV structure ops -------------------*- tablegen -*-===//
```

Comment:
How about "Decl.td" or "TypeDecl.td", since these are AST-like type declarations and the components?

nit: plz fill the comment to 80 columns

Reply:
That sounds good to me. I like "TypeDecl.td" since these (and any future operations) are centered around declaring types.

I'll fix the line length.